x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/execution/search/QuerierTests.java

```diff
@@ -171,4 +184,85 @@ public void testAggSorting_Randomized() {
         });
         assertEquals(expected.subList(0, queueSize), results);
     }
+
+    public Tuple<Integer, Integer> runLocalAggSorterWithNoLimit(int dataSize) {
```

Comment:
No need for this one to be public. Also, the method should be moved last in the class.